.github/workflows/ci-testing.yml

```diff
@@ -108,5 +111,30 @@ jobs:
         uses: actions/upload-artifact@v4
         if: failure()
         with:
-          name: image-failures-${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.shapely-dev }}
-          path: cartopy_test_output-${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.shapely-dev }}
+          name: image-failures-${{ matrix.os }}-${{ matrix.python-version }}
+          path: cartopy_test_output-${{ matrix.os }}-${{ matrix.python-version }}
+
+  # Separate dependent job to only upload one issue from the matrix of jobs
+  create-issue:
+    if: ${{ failure() && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') }}
```

Comment:
Do we want workflow_dispatch to upload an issue as well, or just the scheduled run?

Reply:
Fair point.  I guess if you've done workflow_dispatch then you are actively looking at it and don't need the issue.